jetty9/src/test/resources/structure.yaml

```diff
@@ -29,8 +29,8 @@ fileExistenceTests:
   path: '${jetty.base}/webapps'
   isDirectory: true
   shouldExist: true
-- name: 'app-env.bash exists'
-  path: '/app-env.bash'
+- name: 'setup-env.bash exists'
```

Comment:
I think we should also add a [file contents test](https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/structure_tests/README.md#file-content-tests) to lightly verify that the append happened.